packages/rx_bloc_cli/bin/compile_bundles.sh

```diff
@@ -16,6 +16,11 @@ dart run mason_cli:mason bundle \
   -o lib/src/templates/ \
   mason_templates/bricks/feature_deeplink
 
+dart run mason_cli:mason bundle \
```

Comment:
Same as above

Reply:
the folder name was changed from lib_internationalisation to lib_change_language